src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyle2.java

```diff
@@ -0,0 +1,393 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocstyle;
+
+/*
+ * Config:
+ * scope = private
+ * excludeScope = null
+ * checkFirstSentence = true
+ * endOfSentenceFormat = "([.?!][ \t\n\r\f<])|([.?!]$)"
+ * checkEmptyJavadoc = false
+ * checkHtml = false
```

Comment:
here you have to left only those parameters, that were set in test case

```
        checkConfig.addAttribute("checkFirstSentence", "true");
        checkConfig.addAttribute("checkHtml", "false");
```

same for other input files

Reply:
ok

Reply:
done